.github/workflows/test_arm64.yaml

```diff
@@ -0,0 +1,22 @@
+name: Test ARM64
+permissions: read-all
+on: [push, pull_request]
+jobs:
+  test-linux-arm64-cpu-race:
+    uses: ./.github/workflows/test_template.yaml
+    with:
+      runs-on: actuated-arm64-8cpu-8gb
+      targets: "['linux-arm64-unit-test-4-cpu-race']"
+
+  coverage:
+    needs:
+      - test-linux-arm64-cpu-race
+    runs-on: actuated-arm64-8cpu-8gb
+    steps:
+      - uses: actions/checkout@v4
+      - id: goversion
+        run: echo "goversion=$(cat .go-version)" >> "$GITHUB_OUTPUT"
+      - uses: actions/setup-go@v5
+        with:
+          go-version: "1.21.6"
+      - run: make test
```

Comment:
I don't think we need coverage for ARM64. Running that in the AMD64 workflow should be enough. So, I suggest deleting it.